evalml/model_understanding/graphs.py

```diff
@@ -1526,7 +1526,8 @@ def get_linear_coefficients(estimator, features=None):
             "before using this estimator."
         )
     coef_ = estimator.feature_importance
-    coef_ = pd.Series(coef_, name="Coefficients", index=features)
+    coef_ = pd.Series(coef_, name="Coefficients")
```

Comment:
Do we need all the `pd.Series` calls with this call here?

Reply:
Nope since `coef_` should be a `pd.Series` object now. Though we still want the name, so I can update this line to set the name instead.